src/main/scala/nl/knaw/dans/easy/fedoratobag/DDM.scala

```diff
@@ -38,9 +40,9 @@ object DDM extends DebugEnhancedLogging {
     //    println(new EmdMarshaller(emd).getXmlString)
 
     val dateMap: Map[String, Iterable[Elem]] = getDateMap(emd)
-    val dateCreated = dateMap("created")
+    val dateCreated = dateMap("created").map(d => parseDate(d.text))
     val dateAvailable = {
-      val elems = dateMap("available")
+      val elems = dateMap("available").map(d => parseDate(d.text))
```

Comment:
What about

    ...{ fixDate(value.getValue) }</label>
    ...{ fixDate(value.getDateTime) }</label>
at https://github.com/DANS-KNAW/easy-fedora-to-bag/blob/6c717c885b3420a1320c85e628821b5766afcc1d/src/main/scala/nl/knaw/dans/easy/fedoratobag/DDM.scala#L280-L282